src/NosCore.GameObject/Character.cs

```diff
@@ -394,5 +396,17 @@ public StatPacket GenerateStat()
                 Option = 0
             };
         }
+
+        public SpeakPacket GenerateSpk(SpeakPacket packet)
```

Comment:
if there is a VisualType and a Name in it : it should be in an INamedEntity extension